README.md

```diff
@@ -53,19 +55,19 @@ Before you commit your changes make sure that nothing is broken by running [`yar
 
 #### **5. Commit, push and pull-request**
 
-Now you can commit your changes, make sure to provide a relevant commit message that explains what changes you've made. Then push that commit to your fork, when pushed GitHub actions should run some tests to make sure everything is as it should be and if GitHub is okay you can make a pull request to this repository. Help with all this can be found on GitHub and google.
+Now you can commit your changes, make sure to provide a relevant commit message that explains what changes you've made. Then push that commit to your fork, when pushed GitHub actions should run some tests to make sure everything is as it should be and if GitHub is okay you can make a pull request to this repository. Help with all this can be found on GitHub and Google.
 
 #### **6. Review and merge**
 
 After someone has reviewed your changes they will be merged into the repository and that's it! You've contributed to the IN-Chapters songlist!
 
 ## Contributing (songs)
 
-This section describes how to add to, or edit the songlist. All songs are stored in markdown (`.md`) files following a specific format so they can be read by the builder. Song's filenames should begin with an unique ID (0-4095) and optionally (but recomended) something to identify the song separated by `_`. Generally the identifier will be based on the title of the song, e.g. "Moder Kista" is `0_Moder_Kista.md`. Read more here for the process of how to [add a new song](#creating) and [modifying an existing song](#modifying).
+This section describes how to add to, or edit the songlist. All songs are stored in markdown (`.md`) files following a specific format so they can be read by the builder. Song's filenames should begin with a unique ID (0-4095) and optionally (but recommended) something to identify the song separated by `_`. Generally, the identifier will be based on the title of the song, e.g. "Moder Kista" is `0_Moder_Kista.md`. Read more here for the process of how to [add a new song](#creating) and [modifying an existing song](#modifying).
```

Comment:
TIL it's 'a unique' not 'an unique'

Reply:
“unique” starts with a “u”, but when being pronounced, it makes a “y” sound (yoo-nique), which is a consonant in English. So, “a” should be used as the article.